src/pytest_plugins/test_filler/test_filler.py

```diff
@@ -126,6 +133,14 @@ def pytest_addoption(parser):
         help="Path to dump the transition tool debug output.",
     )
 
+    version_group = parser.getgroup("version_info", "Versioning of the EEST framework")
+    version_group.addoption(
+        "--version-info",
```

Comment:
If we use `--version` it conflicts with the existing flag from pytest

I assumed thats why we used `fill --test-help` instead of `fill --help`

I can look into a workaround